rundeckapp/src/main/groovy/com/dtolabs/rundeck/server/plugins/AppExecutionPluginLoader.groovy

```diff
@@ -51,7 +48,6 @@ class AppExecutionPluginLoader implements IExecutionProviders, ApplicationContex
     @Autowired PluginConfigureService pluginService
     @Autowired IFrameworkNodes rundeckNodeSupport
     @Autowired NodeProviderName nodeProviderName
-    @Autowired RundeckPluginBlocklist rundeckPluginBlocklist
```

Comment:
This also does not seem to be used but I would appreciate someone triple-checking me here...

Reply:
In RundeckproRunnerGrailsPlugin class the property RundeckPluginBlocklist is not loaded in line 214 so probably it would always be null.
Another reference is in rundeckapp/grails-app/conf/spring/resources.groovy line 346 and it is not loaded either.